src/app/(proper_react)/(redesign)/(public)/LandingViewRedesign/components/PrivacyProductBundleBanner.module.scss

```diff
@@ -0,0 +1,154 @@
+@use "../../../../../tokens";
+@use "sass:color";
+
+.banner {
+  align-items: center;
+  background: radial-gradient(
+    141.42% 141.42% at 100% 0%,
+    #b833e11a 0%,
+    #9059ff1a 37.1%,
+    #5b6df81a 61.4%,
+    #0090ed1a 100%
+  );
+  border-radius: tokens.$border-radius-xl;
+  display: flex;
+  flex-direction: column-reverse;
+  font: tokens.$text-body-md;
+  gap: tokens.$spacing-md;
+  justify-content: space-between;
+  overflow: hidden;
+  padding: tokens.$layout-xs;
+  position: relative;
+  width: 100%;
+
+  .content {
+    display: flex;
+    flex-direction: column;
+    gap: tokens.$spacing-lg;
+    max-width: calc(tokens.$content-xs * 1.5);
+  }
+
+  .title {
+    display: block;
+    font: tokens.$text-title-xs;
+    font-weight: 700;
+    text-align: center;
+    width: 100%;
+  }
+
+  .subtitle {
+    display: flex;
+    flex-direction: column;
+    font-weight: 600;
+    gap: tokens.$spacing-xs;
+  }
+
+  .description {
+    display: flex;
+    flex-direction: column;
+    gap: tokens.$spacing-sm;
+    width: 100%;
+  }
+
+  .tags {
+    display: flex;
+    gap: tokens.$spacing-sm;
+
+    .tag {
+      align-items: center;
+      background: hsl(
+        color.hue(tokens.$color-blue-50),
+        color.saturation(tokens.$color-blue-50),
+        87%
+      );
+      border-radius: tokens.$border-radius-xl;
+      display: flex;
+      gap: tokens.$spacing-xs;
+      overflow: hidden;
```

Comment:
Does this do anything? I feel like `hidden` is one of the riskier rules :)

Reply:
It is also actually not needed here, I’ll remove it.